distro_data.json

```diff
@@ -1182,6 +1184,22 @@
             "stdc++": "7.5"
         }
     },
+    "SUSE SLE-15-SP6": {
+        "release": {
+            "date": "2024-06",
+        },
+        "versions": {
+            "kernel": "6.4",
+            "GCC": [ "7.5", "8.2", "9.3", "10.4", "11.3", "12.3", "13.2", "14.2" ],
+            "clang": [ "5.0", "7.0", "9.0", "11.0", "15.0", "17.0" ],
+            "glib": "2.78",
+            "glibc": "2.38",
+            "gtk+3": "3.24",
+            "drm": "2.4.118",
+            "python3": ["3.6", "3.9", "3.10", "3.11", "3.12"],
+            "stdc++": "7.5"
```

Comment:
This is where I'm confused:
```
$ podman run -ti registry.suse.com/suse/sle15:15.6
Trying to pull registry.suse.com/suse/sle15:15.6...
Getting image source signatures
Copying blob 83b67e731066 done  
Copying config 485382c378 done  
Writing manifest to image destination
Storing signatures
b3aa3f8d5c65:/ # rpm -q libstdc++6
libstdc++6-14.2.0+git10526-150000.1.6.1.x86_64
```
(and that's similarly true even on SLE 12SP5, where it's 13.3)
Are these docker images wrong?

Reply:
Huh, actually I had a wrong premise.
gcc7 is the system compiler, and I assumed we ship the libstdc++-version from that, but apparently, we actually update always to the newest version available.
I'll fix this.